cypress.config.ts

```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I updated because on the earlier version there was a problem with starting test:visual. I'll check it another time, but maybe we could stay this changes.